lib/assets/javascripts/cartodb/table/mapview.js

```diff
@@ -877,6 +840,13 @@ cdb.admin.MapTab = cdb.core.View.extend({
 
   },
 
+  getMapView: function() {
+    if (!this.mapView) {
```

Comment:
Interesting. Was this necessary to get things to work?

I would expect that we could safely have this function return null if no mapView, since sometimes we'll be requesting from the table view where there is no map, right?